packages/textarea-sequence/package.json

```diff
@@ -23,5 +23,8 @@
     "lodash-es": "^4.17.11",
     "quill": "1.3.7"
   },
-  "homepage": "https://ebi-webcomponents.github.io/nightingale/"
+  "homepage": "https://ebi-webcomponents.github.io/nightingale/",
+  "devDependencies": {
```

Comment:
That probably needs to be hoisted up to the root of the repo if the tests are run globally